patch_tests.py

```
@@ -0,0 +1,10 @@
+with open('crates/logfwd-io/src/macos_log_input.rs', 'r') as f:
```

Ruff (0.15.11)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)